copy-as-format.el

```diff
@@ -99,12 +99,25 @@ work with your processor add the appropriate mapping here.")
 
 (defconst copy-as-format--jira-supported-languages
   '(("as"  "actionscript")
+    ("adb" "ada")
+    ("ads" "ada")
+    ("cs"  "c#")
+    ("erl" "erlang")
+    ("hs"  "haskel")
     ("htm" "html")
-    ("js"  "javascript")))
-
-(dolist (lang '("html" "java" "sql" "xhtml" "xml"))
+    ("mm"  "objc")
+    ("pl"  "perl")
+    ("pm"  "perl")
+    ("py"  "python")
+    ("rb"  "ruby")
+    ("ksh" "sh")
+    ("vb"  "visualbasic")
+    ("yml" "yaml")))
+
+(dolist (lang '("applescript" "bash" "c" "cpp" "css" "go" "groovy" "html" "java" "javascript" "js" "json" "lua" "none" "nyan" "php" "r" "rainbow"  "scala" "sh" "sql" "swift" "xml" "yaml"))
```

Comment:
What types of files use the `none`, `rainbow` and `nyan` extensions?

Reply:
With `{code:none}`, Jira applies no syntax highlighting to the code block, whereas `{code}` is the same as `{code:java}`.

I had no idea what `rainbow` and `nyan` were, but I've tried them, and they're both the same.  They colour the text with a rainbow pattern.  I don't think that there's any file extension we can reasonably map, here, but I'm minded to leave them in, just for completeness.

Reply:
> With {code:none}, Jira applies no syntax highlighting to the code block...
But `none` is not needed. It's [handled below](https://github.com/sshaw/copy-as-format/pull/3/files#diff-5d26e345e36c2ed8bad6b49d0ee48345R202). 


> I had no idea what rainbow and nyan were... I don't think that there's any file extension we can reasonably map,
If there are no `.nyan` or `.rainbow` file extensions then no need to include that here.